fastify.js

```diff
@@ -6,7 +6,7 @@ const http = require('http')
 const https = require('https')
 const pinoHttp = require('pino-http')
 const Middie = require('middie')
-const fastseries = require('fastseries')
+const runHooks = require('fastseries')()
```

Comment:
can you move this into the `fastify` instance? For perf reasons, we might want to have this encapsulated as well.